### PR TITLE
Add APM FI team to codeowners for plugins

### PR DESCRIPTION
**Pull request by jbertran**
### Plugin Checklist

- [ ] Unit tests.
- [ ] TypeScript [definitions][1].
- [ ] TypeScript [tests][2].
- [ ] API [documentation][3].
- [ ] CircleCI [jobs/workflows][4].
- [ ] Plugin is [exported][5].

[1]: https://github.com/DataDog/dd-trace-js/blob/master/index.d.ts
[2]: https://github.com/DataDog/dd-trace-js/blob/master/docs/test.ts
[3]: https://github.com/DataDog/documentation/blob/master/content/en/tracing/trace_collection/library_config/nodejs.md
[4]: https://github.com/DataDog/dd-trace-js/blob/master/.circleci/config.yml
[5]: https://github.com/DataDog/dd-trace-js/blob/master/packages/dd-trace/src/plugins/index.js

### CODE REVIEW

File: CODEOWNERS
Comment (juan-fernandez) [inline CODEOWNERS:5]: could we exclude CI Visibility specific integrations from this?

File: CODEOWNERS
Comment (juan-fernandez) [inline CODEOWNERS:6]: this folder includes a lot of files, some of which are not exclusive to APM, so I wouldn't include it here

File: CODEOWNERS
Comment (juan-fernandez) [inline CODEOWNERS:7]: same as above

**Review by juan-fernandez — CHANGES_REQUESTED** [review]
I think the amount of files covered is too broad. Also, this means that we won't be able to do releases without `@Datadog/apm-framework-integrations-reviewers-js`, which might be a good thing, but it warrants a broader discussion

File: CODEOWNERS
Comment (labbati) [inline CODEOWNERS:9]: I think we need to add _both_ teams `@DataDog/dd-trace-js` and `@Datadog/apm-framework-integrations-reviewers-js`, otherwise anyone would require _our_ approval to merge anything into those folders, and the JS team's approval alone would not be enough. At least this is how I understand it. If you are sure that this is not the case, feel free to ignore this message.

Ref: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#:~:text=%23%20In%20this%20example%2C%20any%20change%20inside%20the%20%60/scripts%60%20directory%0A%23%20will%20require%20approval%20from%20%40doctocat%20or%20%40octocat.%0A/scripts/%20%40doctocat%20%40octocat

**Comment by juan-fernandez** [comment]
related: https://github.com/DataDog/dd-trace-js/pull/3415

File: CODEOWNERS
Comment (jbertran) [reply CODEOWNERS:9]: 🤦  fixing

File: CODEOWNERS
Comment (jbertran) [reply CODEOWNERS]: Here's a list of the ones I see that should be excluded:

File: CODEOWNERS
Comment (jbertran) [reply CODEOWNERS]: The non-APM ones AFAICT are `ci_plugin` and `log_plugin`. I can leave these to dd-trace-js (you might want to grab ownership of ci_plugin in #3415 anyway)

**Comment by jbertran** [comment]
It would make sense to wait until #3415 merges to have a clearer picture of concern separation inside plugin folders.

**Review by juan-fernandez — APPROVED** [review]

File: CODEOWNERS
Comment (juan-fernandez) [reply CODEOWNERS]: that's correct

**Pull request merged**
